tests/virtual_machines_test.go

```diff
@@ -570,6 +586,14 @@ var _ = Describe("[rfe_id:3503][crit:medium][vendor:cnv-qe@redhat.com][level:com
 					err := testClient.VirtClient.Create(context.TODO(), vm)
 					Expect(err).ToNot(HaveOccurred(), "should success creating the vm")
 				})
+				It("should remove the mac from vmWaitConfigMap after vm created successfully", func() {
```

Comment:
nit: maybe new-line here

Reply:
Adding a new line after `It()` is inconsistent with all the other tests in this module.
why add it here?

Reply:
right. my bad